doc/changelog.rst

```diff
@@ -12,6 +12,7 @@ Unreleased
 Core
 ~~~~
 
+- Add default contact groups
```

Comment:
Be more specific, e.g. "Add contacts automatically to automatically created type specific groups"